scripts/ts-node-register.js

```diff
@@ -1,3 +1,6 @@
-// Register ts-node so that it uses the scripts directory's tsconfig
+// Register ts-node so that it uses the scripts directory's tsconfig and only transpiles
 const tsNode = require('ts-node');
-tsNode.register({ dir: __dirname });
+tsNode.register({
+  dir: __dirname,
+  transpileOnly: true,
```

Comment:
So we will not have any benefits from type checks? Am I correct?

Reply:
I can turn on type checks if you want, I was just concerned about regressing build perf.

Reply:
In general I like type checks aside from the perf concerns :)

Reply:
Turned off type checking again due to perf concerns...nearly 30s per project for `clean`!! 😬